lib/ansible/modules/files/template.py

```diff
@@ -53,6 +53,18 @@
     required: false
     choices: [ "yes", "no" ]
     default: "no"
+  decode:
+    description:
+      - read template using a different encoding (iso8859-15, cp1252)
+    required: false
```

Comment:
If a parameter is not required, you don't have to add `required: false`. It's implicit unless you state `required: true`.